core/src/test/java/edu/wpi/grip/core/MockPipeline.java

```diff
@@ -0,0 +1,25 @@
+
+package edu.wpi.grip.core;
+
+import edu.wpi.grip.core.settings.ProjectSettings;
+
+/**
+ *
+ */
+public class MockPipeline extends Pipeline {
+
+    private ProjectSettings settings;
+
+    public MockPipeline(ProjectSettings settings) {
```

Comment:
You don't actually need this class. There is a "SettingsProvider" class that you can implement that will work.